styleguide/source/assets/scss/01-atoms/_icons.scss

```diff
@@ -43,7 +43,7 @@ $icons: (
 @each $name in $icons {
   .icon--#{$name} {
     display: inline-block;
-    background: url("../../assets/icons/svg/icon-" + $name + ".svg") no-repeat 0 0;
+    background: url("../icons/svg/icon-" + $name + ".svg") no-repeat 0 0;
```

Comment:
All of the icons are no longer appearing now. This pulls in a broken link.

Reply:
hmmm. this was done so in D8 the icon paths were not broken. The icons are not broken on my machine. Perhaps we can compare notes?

Reply:
Did you try the gulp task gulp svg2twig?